content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -54,12 +54,11 @@ In this example:
   In this case, you simply select a label that is defined in the Pod template (`app: nginx`).
   However, more sophisticated selection rules are possible,
   as long as the Pod template itself satisfies the rule.
-
-  {{< note >}}
-  `matchLabels` is a map of {key,value} pairs. A single {key,value} in the `matchLabels` map
-  is equivalent to an element of `matchExpressions`, whose key field is "key", the operator is "In",
-  and the values array contains only "value". The requirements are ANDed.
-  {{< /note >}}
+    {{< note >}}
+    The `matchLabels` is a map of {key,value} pairs. A single {key,value} in the `matchLabels` map
+    is equivalent to an element of `matchExpressions`, whose key field is "key", the operator is "In",
```

Comment:
- You may want to add, The `matchLabels` field is a ...
- It may be helpful to link to the definition of `matchExpressions` here, otherwise the rest of the explanation seems out of context.
- whose `key` field is equal to "key", the `operator` field is set to "In", and the `values` list contains only one item, "value"?
What requirements are ANDed?

Reply:
+1. Thank you! I will update the content.

Reply:
I couldn't find a place to link it. The section was copied from [1]. Therefore, I made sure the content match.

[1] https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements